### PR TITLE
updating pm_con

### PR DESCRIPTION
**Pull request by mohmdrezk**
I was getting an error with model.Steps(iStep).Level
It was overcome by changing to model.Steps{iStep}.Level

### CODE REVIEW

**Comment by mohmdrezk** [comment]
- Another "minor" question: 
The output folder name of the RFX is "trial_type.VisStat"
can we change the name to remove the "trial_type" 
It happens only with the automatic contrasts, but not when comparing 2 conditions.

File: model-motionDecoding_multivariate.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+          ],
+          "weights": [1, -1],
+          "type": "t"
+        }
+      ]
+    },
+    {
+      "Level": "run",
+      "AutoContrasts": ["trial_type.Vis_U", "trial_type.Vis_D"],
+      "Contrasts": []
```
Comment (mohmdrezk) [inline model-motionDecoding_multivariate.json]: will compute the contrast of each event in each run using this level:Run

**Comment by Remi-Gau** [comment]
Would you rather merge this into for now and work to have only one function for FFX and pmCon later?

**Comment by mohmdrezk** [comment]
Yes. We can merge it for now, and I will work on merging them. 
Probably will require an additional input for the function to specify univariate vs multivariate.

**Comment by Remi-Gau** [comment]
Merged from my phone while on a train. You gotta love technology... When it works.

**Comment by Remi-Gau** [comment]
Will update the FFX mvpa file to include the changes the PR from Ane that merges getSession, getSubject...
You might have to do some git magic after that to avoid merge conflicts but let's cross that bridge when we get to it.

**Pull request merged**